lte/gateway/c/core/oai/tasks/amf/include/amf_client_servicer.h

```diff
@@ -60,12 +84,11 @@ class AMFClientServicer : public AMFClientServicerBase {
       task_zmq_ctx_t* task_zmq_ctx_p, task_id_t destination_task_id,
       MessageDef* message_p) override {
     OAILOG_DEBUG(LOG_AMF_APP, " Mock is Enabled \n");
-
+    status_code_e rc = RETURNok;
     itti_free_msg_content(message_p);
     free(message_p);
-    return RETURNok;
+    return rc;
```

Comment:
`rc` is only used once, which can be skipped

```suggestion
    return RETURNok;
```

Reply:
review comment addressed